resources/popup.html

```diff
@@ -30,13 +30,18 @@
             <img id="status_image" class="col s12" src="img/icon/off.png">
         </div>
 
-        <div class="row center">
-            <div class="col s6">
+        <div class="row">
+            <div class="col s4">
                 <a id="status_toggle" class="btn waves-effect lime waves-light">
                     <i class="material-icons">power_settings_new</i>
                 </a>
             </div>
-            <div class="col s6">
+            <div class="col s4">
+                <a class="btn waves-effect lime waves-light" href="../analytics/index.html?userName=thervh70" target="_blank">
+                    <i class="material-icons">timeline</i>
+                </a>
+            </div>
+            <div class="col s4">
```

Comment:
Hmm... the right margin looks smaller than the left margin:
![https://i.gyazo.com/26b374affd73b6b47a1538576f0f7246.png](https://i.gyazo.com/26b374affd73b6b47a1538576f0f7246.png)